generator/templates/cli/registerflag.gotmpl

```diff
@@ -66,6 +66,24 @@ if err := cmd.RegisterFlagCompletionFunc({{ camelize .Name }}FlagName,
     {{- end }}
 {{ end }}
 
+{{ define "arrayregistrator" }}
+    {{- if or (eq .GoType "[]int64") (eq .GoType "[]int32") (eq .GoType "[]string") (eq .GoType "[]float64") (eq .GoType "[]float32") (eq .GoType "[]bool") }}
+        {{ template "flagdescriptionvar" . }}
+        {{ template "flagnamevar" . }}
+        {{ template "flagdefaultvar" . }}
+        _ = cmd.PersistentFlags().{{ pascalize .GoType }}Slice({{ camelize .Name }}FlagName, {{ camelize .Name }}FlagDefault, {{ (camelize .Name) }}Description)
+        {{ template "enumcompletion" . }}
```

Comment:
"enumcompletion" does not support slice type. Maybe you can try 
```
{{ template "enumcompletion" .Item }} 
```
To complete array element

Reply:
The suggested fix didn't work.

enumcompletion is broken for array items even for the primitive types (the code in master is not working). I suggest leaving this for another pull request.

Reply:
You should then remove {{ template "enumcompletion" . }} in those places

Reply:
> You should then remove {{ template "enumcompletion" . }} in those places

Done.